src/phases/scale/LargeScalePhases.yml

```diff
@@ -237,9 +237,9 @@ SnapshotScanner5GigabytesCmd:
   GlobalRate: 1 per 5 minutes
 
 # Leave out the ScanDuration since we'll be supplying the ScanRateMegaBytes option.
-SnapshotScanner5GigabytesFixedRateCmd:
+SnapshotScanner10GigabytesFixedRateCmd:
   Duration: *Duration
-  ScanSizeBytes: 500000000
+  ScanSizeBytes: 10000000000
```

Comment:
I wanted to increase the scan rate (which necessitates increasing the size if we want the same duration).